rider-fsharp/src/test/kotlin/extensions/TestExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package extensions
+
+import com.intellij.openapi.project.Project
+import com.jetbrains.rider.plugins.fsharp.RdExperimentalFeatures
+import com.jetbrains.rider.plugins.fsharp.RdSetFeatures
+import com.jetbrains.rider.plugins.fsharp.rdFSharpModel
+import com.jetbrains.rider.projectView.solution
+import com.jetbrains.rider.test.base.BaseTestWithSolutionBase
+
+fun BaseTestWithSolutionBase.withExperimentalFeatures(project: Project, features: Array<RdExperimentalFeatures>, function: () -> Unit) {
```

Comment:
Please add a test for it, it should check that the feature is actually enabled in the block and is disables afterwards.